### PR TITLE
Rails Form - PR For Review

### PR DESCRIPTION
**Pull request by udberg**
In this project, my partner @Murodjon000  and I @udberg  created a form in rails that can send parameters to a controller so that it can build an instance of a model with those parameters.

### CODE REVIEW

**Comment by Murodjon000** [comment]
### Hi TSE
I and @udberg did some changes in the routes file. In actual User should only have create and new actions but we cannot show the user when he or she signs up so we put show, edit and update actions for this to show the user who signs up and he can also update his profile. We hope you get the meaning.
Thanks for checking our code 😊😊

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root 'users#new'
+  resources :users, only: [:new, :create, :show, :edit, :update]
```
Comment (jubaan) [inline config/routes.rb:4]: - [x] Remove the unnecessary actions, `show` is not going to be used.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      flash[:success] = 'Profile updated'
+      redirect_to @user
+    else
+      render 'edit'
+    end
+  end
```
Comment (jubaan) [inline app/controllers/users_controller.rb]: - [x] `update` should redirect to `edit_user_path` if the user was saved

File: app/controllers/users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+  private
+
+  def user_params
+    params.require(:user).permit(:username, :email, :password, :password_confirmation)
```
Comment (jubaan) [inline app/controllers/users_controller.rb]: - [x] `password_confirmation` is just validation for the `password` so is not really a user parameter, you can safely remove it.

**Review by jubaan — CHANGES_REQUESTED** [review]

**Review by LamiaSristy — APPROVED** [review]
# Status: Approved. 💯 🥇 🎉 🎉

Well done Team!! 👏 👏 👏 👏 

Your project is now approved. Congratulations on your hard work. 🛠️ 


You can merge the branch now, good luck with your next project 🚀.

Feel free to reach me at @lamiasristy 🙋

**Pull request merged**